src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,170 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * The Ui class deals with interactions with users.
+ */
+public class Ui {
+
+
+    private String description;
+
+    public String displayDescription() {
+        return description;
+    }
+
+
```

Comment:
Small issue, but I noticed that line breaks are not consistent in your code. Perhaps you could standardize the line breaks to make the code look cleaner.